deploy/role_binding.yaml

```diff
@@ -1,12 +1,19 @@
-kind: ClusterRoleBinding
-apiVersion: rbac.authorization.k8s.io/v1
-metadata:
-  name: istio-workspace
-subjects:
-- kind: ServiceAccount
-  name: istio-workspace
-  namespace: istio-system
-roleRef:
-  kind: ClusterRole
-  name: istio-workspace
-  apiGroup: rbac.authorization.k8s.io
+kind: Template
```

Comment:
Just wondering - shouldn't we go with upstream k8s first? That would make the adoption a bit wider.